primitives/evm/src/lib.rs

```diff
@@ -29,6 +29,10 @@ use evm::ExitReason;
 pub use evm::backend::{Basic as Account, Log};
 pub use precompile::{Precompile, PrecompileSet, LinearCostPrecompile};
 
+// TODO: should be #[cfg(test)], but that doesn't seem possible
```

Comment:
You can use `#[cfg(all(feature = "std", test))]`

Reply:
See my other comment -- I did try this.